api/config/v1alpha1/envoygateway_types.go

```diff
@@ -258,8 +258,34 @@ type RateLimit struct {
 	// database backend used by the rate limit service to store
 	// state associated with global ratelimiting.
 	Backend RateLimitDatabaseBackend `json:"backend"`
+
+	// XRateLimitHeadersRFCVersion defines the version of the standard to use for X-RateLimit headers.
```

Comment:
I would enable this feature by default without adding any API field.

Reply:
If there is no extreme performance requirement, it is also acceptable to enable it by default.

Reply:
go for it !

Reply:
done